### PR TITLE
DOCS: Removing errors from documentation

### PR DESCRIPTION
**Pull request by HarshCasper**
## Description

This PR serves the purpose of fixing docs issues through the website.

### CODE REVIEW

**Comment by danlester** [comment]
@HarshCasper I also fixed some of these in a recent PR.

I may have approached it differently.

I added 'Keycloak' to the vocab, but not 'keycloak' - and fixed the complaints by using backticks around code, e.g. `security.keycloak`.

And one or two other rules I decided to demote to warnings instead of errors because I didn't feel they add value to our docs, certainly considering they cause tests to fail if broken.

**Review by iameskild — COMMENTED** [review]
Thanks @HarshCasper for all the clean up! I have added a few suggestions based on my review. The two most suggestions are converting `It's` to `it's` when in the middle of a sentence and adding back the `s` in `k9s`.

File: docs/source/dev_guide/minikube.md [hunk 1]
```diff
@@ -29,35 +19,29 @@ This guide assumes that you have the QHub repository downloaded, and you are at
 > NOTE: The following instructions apply **only to Linux OS**.
 
 To deploy QHub locally requires the installation of the following dependencies:
-+ [Minikube](https://v1-18.docs.kubernetes.io/docs/tasks/tools/install-minikube/) version 1.10.0-beta and up
-+ [Docker Engine driver](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository) install.
 
-The installation of a hypervisor is **not** necessary.
+- [Minikube](https://v1-18.docs.kubernetes.io/docs/tasks/tools/install-minikube/) version 1.10.0-beta and up
```
Comment (iameskild) [inline docs/source/dev_guide/minikube.md:23]:
```suggestion
- [Minikube](https://minikube.sigs.k8s.io/docs/start/) version 1.10.0-beta and up
```

File: docs/source/dev_guide/minikube.md [hunk 2]
```diff
@@ -29,35 +19,29 @@ This guide assumes that you have the QHub repository downloaded, and you are at
...
+- [Docker Engine driver](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository) install.
+
+The installation of a hypervisor isn't necessary.
 
 > NOTE: Minikube requires `kubectl` OR you can use the embedded kubectl appropriate for your Minikube cluster version using `minikube kubectl`.
 > To install `kubectl` [follow the instructions](https://v1-18.docs.kubernetes.io/docs/tasks/tools/install-kubectl/) according to your operating system.
```
Comment (iameskild) [inline docs/source/dev_guide/minikube.md:29]:
```suggestion
> To install `kubectl` [follow the instructions](https://kubernetes.io/docs/tasks/tools/) according to your operating system.
```

**Review by iameskild — APPROVED** [review]

**Pull request merged**